ignite/metrics/epoch_metric.py

```diff
@@ -136,7 +136,7 @@ def update(self, output: Tuple[torch.Tensor, torch.Tensor]) -> None:
             except Exception as e:
                 warnings.warn(f"Probably, there can be a problem with `compute_fn`:\n {e}.", EpochMetricWarning)
 
-    def compute(self) -> float:
+    def compute(self) -> Any:
```

Comment:
I’m not sure about this. Returned value is float.

Reply:
As I am over riding the function and the return type of the over ridden function in precision_recall_curve class is Tuple which is not a sub class of float that is why I changed it to 'Any'. You can see this once as well https://github.com/pytorch/ignite/runs/5408929040?check_suite_focus=true#step:12:12

Apart from this if it is better to cast the return value of compute function in precision_recall_curve class to float I will do that.